.env.test

```diff
@@ -1,6 +1,8 @@
 DEBUG=true
 SECRET_KEY='tests'
 DATABASE_URL='postgres://localhost/opencraft'
+INSTANCE_MYSQL_URL='mysql://root@localhost'
+INSTANCE_MONGO_URL='mongodb://localhost'
```

Comment:
@omarkhan This causes to require MySQL and Mongo to be installed locally on the instance VM for the tests to pass - while it seem right to require the instance manager's own DB to be accessible for tests, the DBs used by instances themselves are a different type, and wouldn't be expected to be on the instance manager's own VM. If we add other instance types in the future, which could themselves require other DB types, that could make the instance manager VM environment pretty unwieldy.

Looking at the tests that use this below, the ones that require this look more like integration tests - would it make sense to move the tests that require these DBs to be setup to integration tests?